packages/client/modules/userDashboard/components/OrgBilling/OrgBilling.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import {createFragmentContainer} from 'react-relay'
+import {createRefetchContainer} from 'react-relay'
```

Comment:
add `RelayRefetchProp` and you can use that as your type. then relay.refetch will be guaranteed nonnull

Reply:
Nice! I'll use that instead